wordpress-mobile/gutenberg-mobile
# Update gutenberg ref

This is a fast PR that brings the gutenberg ref back to master.

To test:
- yarn clean:install
- Run the app
- Smoke test around to check that things look fine

Update release notes:
